arbnode/seq_coordinator.go

```diff
@@ -55,18 +55,19 @@ type SeqCoordinator struct {
 }
 
 type SeqCoordinatorConfig struct {
-	Enable                bool                       `koanf:"enable"`
-	ChosenHealthcheckAddr string                     `koanf:"chosen-healthcheck-addr"`
-	RedisUrl              string                     `koanf:"redis-url"`
-	LockoutDuration       time.Duration              `koanf:"lockout-duration"`
-	LockoutSpare          time.Duration              `koanf:"lockout-spare"`
-	SeqNumDuration        time.Duration              `koanf:"seq-num-duration"`
-	UpdateInterval        time.Duration              `koanf:"update-interval"`
-	RetryInterval         time.Duration              `koanf:"retry-interval"`
-	SafeShutdownDelay     time.Duration              `koanf:"safe-shutdown-delay"`
-	MaxMsgPerPoll         arbutil.MessageIndex       `koanf:"msg-per-poll"`
-	MyUrlImpl             string                     `koanf:"my-url"`
-	Signing               signature.SignVerifyConfig `koanf:"signer"`
+	Enable                 bool                       `koanf:"enable"`
+	ChosenHealthcheckAddr  string                     `koanf:"chosen-healthcheck-addr"`
+	RedisUrl               string                     `koanf:"redis-url"`
+	LockoutDuration        time.Duration              `koanf:"lockout-duration"`
+	LockoutSpare           time.Duration              `koanf:"lockout-spare"`
+	SeqNumDuration         time.Duration              `koanf:"seq-num-duration"`
+	UpdateInterval         time.Duration              `koanf:"update-interval"`
+	RetryInterval          time.Duration              `koanf:"retry-interval"`
+	ShutdownHandoffTimeout time.Duration              `koanf:"shutdown-handoff-timeout"`
```

Comment:
I think we can re-use SafeShutdownDelay for this. It's a little less flexible, but should be good enough and less configs is better.. your call.

Reply:
Discussed on call, but I've decided not to do this